src/internal/operations/retention_config.go

```diff
@@ -0,0 +1,77 @@
+package operations
+
+import (
+	"context"
+	"time"
+
+	"github.com/alcionai/clues"
+
+	"github.com/alcionai/corso/src/internal/common/crash"
+	"github.com/alcionai/corso/src/internal/events"
+	"github.com/alcionai/corso/src/internal/kopia"
+	"github.com/alcionai/corso/src/internal/stats"
+	"github.com/alcionai/corso/src/pkg/control"
+	"github.com/alcionai/corso/src/pkg/control/repository"
+	"github.com/alcionai/corso/src/pkg/count"
+)
+
+// RetentionConfigOperation wraps an operation with restore-specific props.
+type RetentionConfigOperation struct {
+	operation
+	Results RetentionConfigResults
+	rcOpts  repository.Retention
```

Comment:
Should we add a level of indirection here? You might be a better judge of the decision, but just wanted to float the idea out there.

Reply:
I think we'll be alright (famous last works 😂). The struct is meant to just carry data without functionality. It will also allow us to add more fields if needed without changing the API